### ISSUE TITLE
Adding a Json2Lang option?

### ISSUE THREAD

**Issue by vinivosh** [ichttt/MCLang2Json#8]
I'm trying to convert a newer minecraft translation (1.16.3) to Minecraft Bedrock (1.16.100). Seems like Bedrock edition unfortunately uses Lang still. This converter would be perfect if it worked the other way around 😛

**Comment by NuruddinPlays** [comment]
Agree! I wanted to backport a language pack project to older versions that uses Lang, but I can't find any... So this would be a really great option to have!
